src/main/java/net/countercraft/movecraft/combat/features/tracking/BlockBehaviorOverride.java

```diff
@@ -0,0 +1,420 @@
+                } catch (NumberFormatException | NullPointerException ex) {
+                    MovecraftCombat.getInstance().getLogger()
+                            .warning("Unable to load " + m.name() + ": " + entry.getValue());
+                    continue;
+                }
+                addToSet.apply(m);
+                putFunction.accept(m, value);
+            }
+        }
+    }
+
+    public static void enable() {
+        processOverriddenEntry(BlockBehaviorOverride::set, "Failed to set block overrides!");
+    }
+
+    public static void disable() {
+        processOverriddenEntry(BlockBehaviorOverride::reset, "Failed to revert block overrides to vanilla!");
+    }
+
+    protected static void processOverriddenEntry(final BiFunction<Material, BlockOverride, Boolean> function, final String msgOnException) {
+        try {
+            for (Map.Entry<Material, BlockOverride> entry : BLOCK_OVERRIDES.entrySet()) {
+                if (!function.apply(entry.getKey(), entry.getValue()))
+                    MovecraftCombat.getInstance().getLogger().warning("Unable to set " + entry.getKey().name());
+            }
+        } catch (Exception e) {
+            MovecraftCombat.getInstance().getLogger().info(msgOnException);
+            e.printStackTrace();
+        }
+    }
+
+    protected static boolean set(Material mat, BlockOverride override) {
+        boolean result = true;
+
+        // Blast resistance
+        if (override.blastResistanceOverride().isPresent()) {
+            result = result && NMS_HELPER.setBlastResistance(mat, override.blastResistanceOverride().get().floatValue());
+        }
+        // Burn oddity
+        if (override.burnOddity().isPresent()) {
+            result = result && NMS_HELPER.setBurnOdds(mat, override.burnOddity().get().intValue());
+        }
+        // Ignite oddity
+        if (override.igniteOddity().isPresent()) {
+            result = result && NMS_HELPER.setIgniteOdds(mat, override.igniteOddity().get().intValue());
+        }
+
+        return result;
+    }
+
+    protected static boolean reset(Material mat, BlockOverride override) {
+        boolean result = true;
+
+        // Blast resistance
+        if (override.blastResistanceOverride().isPresent() && override.vanillaBlastResistanceOverride().isPresent()) {
+            result = result && NMS_HELPER.setBlastResistance(mat, override.vanillaBlastResistanceOverride().get().floatValue());
+        }
+        // Burn oddity
+        if (override.burnOddity().isPresent() && override.vanillaBurnOddity().isPresent()) {
+            result = result && NMS_HELPER.setBurnOdds(mat, override.vanillaBlastResistanceOverride().get().intValue());
+        }
+        // Ignite oddity
+        if (override.igniteOddity().isPresent() && override.vanillaIgniteOddity().isPresent()) {
+            result = result && NMS_HELPER.setIgniteOdds(mat, override.vanillaBlastResistanceOverride().get().intValue());
+        }
```

Comment:
You can use `result &= _` for these, and there's no need to unbox the values with `.floatValue()` or `.intValue()`.